app/views/content/what-pension-does-a-teacher-get.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "What pension does a teacher get?"
+heading: "What pension does a teacher get?"
+description: |-
+  The teachers' pension is one of the most generous in the country. Learn more about the benefits of the pension scheme once you become a teacher.
+date: "2022-11-29"
+image: false
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+related_content:
+    Salaries and benefits : "/salaries-and-benefits"
+    Steps to become a teacher : "/steps-to-become-a-teacher"
+    Train to be a teacher if you have or are studying for a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Scholarships and bursaries for training to teach : "/funding-and-support/scholarships-and-bursaries"
+keywords:
+  - Teachers pension
+  - Pension
+  - Benefits
+  - Earnings
+  - Teaching
+  - Teacher training
+  - How to become a teacher
+  - Teacher training courses
+
+---
+One of the great benefits of a teaching career is a secure pension that will help you save for your future. 
+
+The teachers’ pension scheme is one of the most generous in the country. These are some of the key benefits of the scheme.
+
+## Easy to get started
+
+As a teacher, you’ll be automatically enrolled at the start of your career, and remain in the scheme when moving teaching jobs, unless you opt out. 
+
+Under the scheme, an amount from your pay is automatically paid into your pension each month. At the same time, the employer also contributes.
```

Comment:
(might just be being picky - sorry!) does 'an amount from your pay' sound a bit too around the houses?